inc/ui/namespace.php

```diff
@@ -4,61 +4,69 @@
-	</g>
-</svg>
-END;
+use WP_Admin_Bar;
 
 function bootstrap() {
 	add_action( 'admin_menu', __NAMESPACE__ . '\\register_menu' );
+	add_action( 'admin_bar_menu', __NAMESPACE__ . '\\admin_bar_menu', 11 );
 }
 
 function register_menu() {
-	$hook = add_menu_page(
-		'Human Made Platform',
-		'Platform',
+	$hook = add_submenu_page(
+		null,
+		'Documentation',
+		'Documentation',
 		'edit_posts',
-		'hm-platform',
-		__NAMESPACE__ . '\\render_page',
-		'data:image/svg+xml;base64,' . base64_encode( LOGO ),
-		2
+		'hm-platform-documentation',
+		__NAMESPACE__ . '\\render_page'
 	);
 
 	add_action( sprintf( 'load-%s', $hook ), __NAMESPACE__ . '\\load_page' );
 }
 
+function admin_bar_menu( WP_Admin_Bar $wp_admin_bar ) {
+	// Add WordPress.org link
+	$wp_admin_bar->add_menu( [
+		'parent'    => 'hm-platform-logo',
+		'id'        => 'documentation',
+		'title'     => 'Documentation',
+		'href'      => add_query_arg( 'page', 'hm-platform-documentation', admin_url( 'admin.php' ) ),
+	] );
+}
+
 function load_page() {
-	wp_enqueue_style( __NAMESPACE__, plugins_url( '/assets/style.css', Documentation\DIRECTORY . '/wp-is-dumb' ) );
-	wp_enqueue_style( 'highlightjs', 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/default.min.css' );
+	wp_enqueue_style( 'highlightjs', 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/vs2015.min.css' );
 	wp_enqueue_script( 'highlightjs', 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js' );
+	wp_enqueue_script( 'highlightjs-line-numbers', 'https://cdn.jsdelivr.net/npm/highlightjs-line-numbers.js@2.7.0/dist/highlightjs-line-numbers.min.js' );
+
+	wp_enqueue_style( __NAMESPACE__, plugins_url( '/assets/style.css', Documentation\DIRECTORY . '/wp-is-dumb' ) );
```

Comment:
That's a ryan special!

Reply:
😎

This is because `plugins_url` needs the plugin file, but we're only storing the directory. Instead of adding another constant, I just added some dummy text so that `dirname` has something to eat.